webapi/CopilotChat/Controllers/ChatController.cs

```diff
@@ -116,7 +116,7 @@ public ChatController(ILogger<ChatController> logger, ITelemetryService telemetr
                 return this.BadRequest(string.Concat(aiException.Message, " - Detail: " + aiException.Detail));
             }
 
-            return this.BadRequest(result.LastErrorDescription);
+            return this.BadRequest(result.LastException!.Message);
```

Comment:
Do we know for a fact that LastException cannot be null?  If not, maybe something like this might be in order:

```suggestion
            return this.BadRequest(result.LastException?.Message);
```

Reply:
ErrorOccurred evalutes to LastException != null

I think this is moving in the right direction, but if feels like a lower energy level could still be attained.